Platform/ApollolakeBoardPkg/AcpiTables/Dsdt/Sc.asl

```diff
@@ -106,8 +106,9 @@ scope (\_SB.PCI0) {
         Return (0x00140000)
       }
     }
+    Name(_PRF, 0x09)
```

Comment:
Can we use a different name?   
Keyword starts with "_" is ACPI spec reserved and should be avoided.

Reply:
Sure. Any preference?

Reply:
How about PEWK ?